### PR TITLE
docs: migrating https://github.com/eclipse-che/che-docs/pull/2202/files

### PR DESCRIPTION
**Pull request by max-cx**
## What does this pull request change
docs: migrating https://github.com/eclipse-che/che-docs/pull/2202/files

The heading 'Managing users' seems to be the best choice.
In contrast to the old docs, this chapter should cover more topics than just identities and authorization in the old docs.
https://docs.microsoft.com/en-us/windows/win32/ad/managing-users
https://docs.microsoft.com/en-us/azure/industry/training-services/microsoft-community-training/user-management/user-management-overview
https://docs.microsoft.com/en-us/services-hub/unified/contracts/manage-users

## What issues does this pull request fix or reference
RHDEVDOCS-3501

## Pull request checklist

The author and the reviewers validate the content of this pull request with the following checklist, in addition to the [automated tests](code_review_checklist.adoc).

- Any procedure:
  - [ ] Successfully tested.
- Any page or link rename:
  - [ ] The page contains a redirection for the previous URL.
  - Propagate the URL change in:
    - [ ] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/packages/dashboard-frontend/src/services/bootstrap/branding.constant.ts)
    - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/master/src/constants.ts)
- [ ] Builds on [Eclipse Che hosted by Red Hat](https://workspaces.openshift.com).
- [ ] the *`Validate language on files added or modified`* step reports no vale warnings.

### CODE REVIEW

**Comment by max-cx** [comment]
@xbaran4, could you please review this PR?

File: modules/administration-guide/partials/proc_configuring-gitlab-oauth2.adoc [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+.Prerequisites
+
+* The GitLab server is running and available for {prod-short}.
+
+.Procedure
+
+. Follow GitLab's instructions for link:https://docs.gitlab.com/ee/integration/oauth_provider.html[configuring GitLab as an OAuth 2.0 authentication identity provider] to create an OAuth2 link:https://docs.gitlab.com/ee/integration/oauth_provider.html#authorized-applications[Authorized application] in Gitlab.
+
+. Enter {prod-short} as the application `Name`.
```
Comment (max-cx) [inline modules/administration-guide/partials/proc_configuring-gitlab-oauth2.adoc:14]: @xbaran4, in your original, this sentence began with "Enter Che as the application `Name`". I replaced `Che` with `{prod-short}`: is this still correct? Can the downstream docs say "Enter `Codeready Workspaces` as the application `Name`? Or should I stick to `Che` here?

Comment (xbaran4) [reply modules/administration-guide/partials/proc_configuring-gitlab-oauth2.adoc:14]: I checked if Gitlab allows for multi-word application name and it does. So your change is correct.

**Review by xbaran4 — APPROVED** [review]

File: modules/administration-guide/partials/proc_configuring-gitlab-oauth2.adoc [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+. Enter the {prod-short} OAuth callback URL as the `Redirect URI`. The default value is `++https://++{prod-deployment}-{prod-namespace}.__<domain>__/api/oauth/callback`.
+
+. Check the `Confidential` and `Expire access tokens` checkboxes.
```
Comment (swatikarmarkar) [inline modules/administration-guide/partials/proc_configuring-gitlab-oauth2.adoc:18]:
```suggestion
. Select `Confidential` and `Expire access tokens`.
```

File: modules/administration-guide/partials/proc_configuring-gitlab-oauth2.adoc [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+. Under *Scopes*, check the `api`, `write_repository`, and `openid` checkboxes.
```
Comment (swatikarmarkar) [inline modules/administration-guide/partials/proc_configuring-gitlab-oauth2.adoc:20]:
```suggestion
. Under *Scopes*, select `api`, `write_repository`, and `openid`.
```

**Comment by themr0c** [comment]
Outdated.

**Pull request closed without merging**